Formula/cayley.rb

```diff
@@ -1,89 +1,16 @@
-  url "https://github.com/cayleygraph/cayley.git",
-    :tag      => "v0.7.7",
-    :revision => "dcf764fef381f19ee49fad186b4e00024709f148"
+  desc "An open-source graph database"
+  homepage "An open-source graph database"
+  url "https://github.com/cayleygraph/cayley/releases/download/v0.7.7/cayley_0.7.7_darwin_amd64.tar.gz"
+  sha256 "37ef9043cb20aeffa93d57f360fdee3e0c9d1b2fa8d404a00f3bff1b58937206"
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "e647be34623b1a8d635df7508f09111dfd8eb6f368a6979f9c5619b016beda8c" => :mojave
-    sha256 "4177fdcb60422d484f1377e5367844ebcc9be471fffc76e717d8ad90e49ee99c" => :high_sierra
-  end
-
-  depends_on "bazaar" => :build
-  depends_on "go" => :build
-  depends_on "mercurial" => :build
+  bottle :unneeded
 
   def install
-    ENV["GOPATH"] = buildpath
-
-    dir = buildpath/"src/github.com/cayleygraph/cayley"
-    dir.install buildpath.children
-
-    cd dir do
-      commit = Utils.popen_read("git rev-parse --short HEAD").chomp
-
-      ldflags = %W[
-        -s -w
-        -X github.com/cayleygraph/cayley/version.Version=#{version}
-        -X github.com/cayleygraph/cayley/version.GitHash=#{commit}
-      ]
-
-      system "go", "build", "-o", bin/"cayley", "-ldflags", ldflags.join(" "), ".../cmd/cayley"
-
-      inreplace "cayley_example.yml", "./cayley.db", var/"cayley/cayley.db"
-      etc.install "cayley_example.yml" => "cayley.yml"
-
-      # Install samples
-      system "gzip", "-d", "data/30kmoviedata.nq.gz"
-      (pkgshare/"samples").install "data/testdata.nq", "data/30kmoviedata.nq"
-    end
-  end
-
-  def post_install
-    unless File.exist? var/"cayley"
-      (var/"cayley").mkpath
-
-      # Initialize the database
-      system bin/"cayley", "init", "--config=#{etc}/cayley.yml"
-    end
-  end
-
-  plist_options :manual => "cayley http --config=#{HOMEBREW_PREFIX}/etc/cayley.conf"
-
-  def plist; <<~EOS
-    <?xml version="1.0" encoding="UTF-8"?>
-    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-    <plist version="1.0">
-      <dict>
-        <key>KeepAlive</key>
-        <dict>
-          <key>SuccessfulExit</key>
-          <false/>
-        </dict>
-        <key>Label</key>
-        <string>#{plist_name}</string>
-        <key>ProgramArguments</key>
-        <array>
-          <string>#{opt_bin}/cayley</string>
-          <string>http</string>
-          <string>--config=#{etc}/cayley.conf</string>
-        </array>
-        <key>RunAtLoad</key>
-        <true/>
-        <key>WorkingDirectory</key>
-        <string>#{var}/cayley</string>
-        <key>StandardErrorPath</key>
-        <string>#{var}/log/cayley.log</string>
-        <key>StandardOutPath</key>
-        <string>#{var}/log/cayley.log</string>
-      </dict>
-    </plist>
-  EOS
+    bin.install "cayley"
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/cayley version")
+    system "#{bin}/cayley", "version"
```

Comment:
Please improve the test, don't make it worse

Reply:
I'll revert that.